frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.tolerations.scenario.ts

```diff
@@ -28,22 +29,22 @@ describe('KubeVirt VM detail - edit Tolerations', () => {
       await click(editTolerationsView.addLabelBtn);
       await editTolerationsView.tolerationKeyInputByID(0).sendKeys('key');
       await editTolerationsView.tolerationValueInputByID(0).sendKeys('value');
-      await click(editTolerationsView.submitBtn);
+      await click(saveButton);
 
       await browser.wait(
         until.textToBePresentInElement(
           virtualMachineView.vmDetailTolerations(vm.namespace, vm.name),
-          'key=value',
+          '1 Tolerations rules',
         ),
       );
 
       await vm.modalEditTolerations();
       await click(editTolerationsView.deleteBtnByID(0));
-      await click(editTolerationsView.submitBtn);
+      await click(saveButton);
       await browser.wait(
         until.textToBePresentInElement(
           virtualMachineView.vmDetailTolerations(vm.namespace, vm.name),
-          'No selector',
+          'No tolerations',
```

Comment:
It seems to me that tolerations should be written with capical T, because when there is a toleration, the message is
`1 Tolerations rules`?
@gouyang Could you file a bug for it and assert 'No Tolerations' instead?

Reply:
Good point, filed bug https://bugzilla.redhat.com/show_bug.cgi?id=1839732